web/src/app/pages/profile/profilePage.controller.js

```diff
@@ -46,65 +46,45 @@
             fileInput.click();
         };
 
-        var getDetails = function() {
-            var name = AuthenticationService.getUser();
-
-            var details = {
-                "id": name
-            };
-
-            $http({
-                method: 'POST',
-                url: IG.api + 'users/getUser',
-                headers: {
-                    'Content-Type': 'application/json'
-                },
-                data: details
-            }).then(function successCallback(response) {
 
-                $scope.data = {};
-                $scope.data = response.data.data.Item;
-
-                if (response.data.data.Item.social == undefined) {
-                    $scope.socialProfiles = [{
-                        name: 'Facebook',
-                        icon: 'socicon-facebook'
-                    }, {
-                        name: 'LinkedIn',
-                        icon: 'socicon-linkedin'
-                    }, {
-                        name: 'GitHub',
-                        icon: 'socicon-github'
-                    }, {
-                        name: 'StackOverflow',
-                        icon: 'socicon-stackoverflow'
-                    }];
-                } else {
-                    $scope.socialProfiles = response.data.data.Item.social;
-                }
-
-                if (response.data.data.Item.techs == undefined) {
-                    $scope.techs = [{
-                            "id": 1,
-                            "name": "Angular"
 
-                        }, {
-                            "id": 2,
-                            "name": "React"
+        $scope.data = user;
+        // console.log(user);
 
-                        }
+        if (user.social == undefined) {
+            $scope.socialProfiles = [{
+                name: 'Facebook',
+                icon: 'socicon-facebook'
+            }, {
+                name: 'LinkedIn',
+                icon: 'socicon-linkedin'
+            }, {
+                name: 'GitHub',
+                icon: 'socicon-github'
+            }, {
+                name: 'StackOverflow',
+                icon: 'socicon-stackoverflow'
+            }];
+        } else {
+            $scope.socialProfiles = user.social;
+        }
+
+        if (user.techs == undefined) {
+            $scope.techs = [{
```

Comment:
Consider moving the defaults to a constant to avoid conditionals.
Example:
JS
```
var techstack = [{ ... }, { ... }, ... ];

$scope.techs = user.techs || techstack;
```